benchmark/benchmarkTests.go

```diff
@@ -82,15 +82,17 @@ func SociRPullPullImage(
 
 func SociFullRun(
 	b *testing.B,
+	ctx context.Context,
 	imageRef string,
 	indexDigest string,
-	readyLine string) {
-	ctx, cancelCtx := framework.GetTestContext()
+	readyLine string,
+	testName string) {
+	log.G(ctx).WithField("test_name", testName).WithField("benchmark", "Test").WithField("event", "Start").Infof("Start Test")
```

Comment:
Just a single test log line for now.  Future PRs will have more log lines.  The rough plan is to have start and stop log lines for different lifecycle events